sematic/plugins/building/BUILD

```diff
@@ -1,6 +1,6 @@
 sematic_py_lib(
     name = "docker_builder",
-    srcs = ["docker_builder.py"],
+    srcs = ["docker_builder.py", "docker_client_utils.py"],
```

Comment:
We should probably stick to our standard pattern of a 1:1 mapping between `sematic_py_lib`s and files. So `docker_client_utils` should be its own target, with `docker_builder` depending on it.

Reply:
Done.